### PR TITLE
ref(arithmetic): Hiding equation aliases from the columns

### PR DESCRIPTION
**Pull request by wmak**
- Instead of `equation[0]` we'll show the equation truncated to 60 characters, with a tooltip on hover
- Not using overflowEllipsis since we only want to show the tooltip when the header is too long

# Preview
image

### CODE REVIEW

**Review by shruthilayaj — APPROVED** [review]

**Comment by shruthilayaj** [comment]
Minor, but not sure if the title needs to be aligned to the right if the column width is wider than the title
Screen Shot 2021-06-29 at 12 08 29 PM

File: static/app/views/eventsV2/table/tableView.tsx [hunk 1]
```diff
@@ -200,22 +200,22 @@ class TableView extends React.Component<TableViewProps> {
...
+    const title = isEquationAlias(column.name)
+      ? eventView.getEquations()[getEquationAliasIndex(column.name)]
+      : column.name;
+    const shouldTruncate = title.length > 60;
 
     const sortLink = (
       <SortLink
         align={align}
-        title={column.name}
+        title={title.substring(0, 60) + (shouldTruncate ? '...' : '')}
```
Comment (k-fish) [inline static/app/views/eventsV2/table/tableView.tsx:211]: Can you just wrap this in a truncate instead maybe? Handrolling substrings feels a bit meh.

**Review by k-fish — APPROVED** [review]

File: static/app/views/eventsV2/table/tableView.tsx (hunk 1 above)
Comment (Zylphrex) [inline static/app/views/eventsV2/table/tableView.tsx:211]: Is this substring necessary? This should already be truncating via CSS.

Comment (k-fish) [reply static/app/views/eventsV2/table/tableView.tsx:211]: Beat you to it Tony.

**Comment by wmak** [comment]
> Minor, but not sure if the title needs to be aligned to the right if the column width is wider than the title

:+1: Good catch, had to rearrange the elements a bit to get this to work properly

**Pull request merged**